docs/getting-started/introduction.md

```diff
@@ -24,31 +24,31 @@ It is designed from the ground up to support Next.js and Serverless.
 
 NextAuth.js can be used with or without a database.
 
-- An open source solution that allows you to keep control of your data
+- An open-source solution that allows you to keep control of your data
 - Supports Bring Your Own Database (BYOD) and can be used with any database
 - Built-in support for [MySQL, MariaDB, Postgres, SQL Server, MongoDB and SQLite](/configuration/databases)
 - Works great with databases from popular hosting providers
 - Can also be used _without a database_ (e.g. OAuth + JWT)
 
-_Note: Email sign in requires a database to be configured to store single-use verification tokens._
+_Note: Email sign-in requires a database to be configured to store single-use verification tokens._
 
 ### Secure by default
 
-- Promotes the use of passwordless sign in mechanisms
-- Designed to be secure by default and encourage best practice for safeguarding user data
-- Uses Cross Site Request Forgery Tokens on POST routes (sign in, sign out)
+- Promotes the use of a passwordless sign-in mechanisms
```

Comment:
This cannot be right.

Reply:
Yeah this would be right only if mechanism*s* is changed to singular, so:
```
- Promotes the use of a passwordless sign-in mechanism
```